File: FlightsAngularNet/ClientApp/src/app/api/models/flights-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from './time-place-rm';
export interface FlightsRm {
  airline?: null | string;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  id?: string;
  price?: null | string;
  remainingNumberOfSeats?: number;
}

File: FlightsAngularNet/ReadModels/FlightsRm.cs
namespace FlightsAngularNet.ReadModels
{
    public record FlightsRm(
        Guid Id,
        String Airline,
        String Price,
        TimePlaceRm Departure,
        TimePlaceRm Arrival,
        int RemainingNumberOfSeats
        
        );
   
}
